src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewModel.swift

```diff
@@ -227,6 +264,24 @@ final class HealthCertificateReissuanceConsentViewModel {
 		}
 	}
 	
+	private func filterAccompanyingCertificates(
+		certificates: [DCCCertificateContainerExtended],
+		certifiedPerson: HealthCertifiedPerson
+	) -> [HealthCertificate] {
+		var finalArray = [DCCCertificateContainer]()
+		let reissuanceCertificates = certificates.compactMap({ $0.certificateToReissue })
```

Comment:
```suggestion
		let reissuanceCertificates = certificates.map { $0.certificateToReissue }
```
Looks like there is no optionality involved here